src/index.ts

```diff
@@ -35,7 +34,7 @@ export class Route {
     return new Route(parts, route.query)
   }
   toString(encode: boolean = true): string {
-    const nonNullQuery = records.filter(this.query, part => part !== undefined && part !== null)
+    const nonNullQuery = records.filter(this.query, part => part !== undefined)
```

Comment:
@Eldow the naming is now wrong (nonUndefinedQuery)?

Reply:
oh that's right